airbyte_ui_rock/rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: airbyte-ui
+summary: Airbyte UI rock
+description: Airbyte UI OCI image for the Airbyte UI charm
+version: "1.0"
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  airbyte-webapp:
+    override: replace
+    summary: "airbyte-webapp service"
+    startup: disabled
+    command: "/usr/bin/pnpm -C airbyte-webapp start"
+    environment:
+      PORT: "8080"
+
+parts:
+  patches:
```

Comment:
suggestion: we ended up removing this from trino and ranger and using a forked repo instead for easier upgrades. You might want to consider this also.

Reply:
I think i'll consider this once there's a large drift between the upstream and the fork.